src/bramble/client/main.js

```diff
@@ -122,7 +122,34 @@ define([
 
     // Expose Filer for Path, Buffer, providers, etc.
     Bramble.Filer = Filer;
-    var _fs = new Filer.FileSystem();
+    var _fs;
+
+    // Set URL param for memory-backed FileSystem
+    var mem_filesystem = "BrambleMemoryFileSystem";
+
+    // Splits the parameter from the URL
+    function getUrlParams() {
+      var param = decodeURIComponent( window.location.href.slice( window.location.href.indexOf( '?' ) + 1 ) );
```

Comment:
You can use `window.location.search` instead of doing this work to slice/indexOf, see https://developer.mozilla.org/en-US/docs/Web/API/Location

Reply:
this probably also doesn't need to be a function and can just be a variable.